tox.ini

```diff
@@ -43,7 +43,7 @@ commands = pre-commit run --all-files
 commands =
     pflake8
     pylint {toxinidir}/src/
-    pylint {toxinidir}/tests/ --disable=E0401,W0212,W0621
+    pylint {toxinidir}/tests/ --disable=E0401,W0212,W0621,R0914
```

Comment:
It's OK for tests to have too many local variables.

Reply:
Just want to make a note here: this is temporary change to keep the CI happy. This will be updated with the latest templates in PR #35